### ISSUE TITLE
Error when loading a Timeline example in PhantomJS

### ISSUE THREAD

**Issue by josdejong** [almende/vis#1170]
For some reason PhantomJS fails to render a Timeline. 

Try the following script:

```js
// test.js
var page = require('webpage').create();
page.open('http://visjs.org/examples/timeline/basicUsage.html', function(status) {
  console.log("Status: " + status);
  if(status === "success") {
    page.render('example.png');
  }
  phantom.exit();
});
```

When running PhantomJS, it throws an error and the created example.png is empty:

```
$ phantomjs test.js
TypeError: 'undefined' is not a function (evaluating 'this.redraw.bind(this)')

  http://visjs.org/dist/vis.js:15142
  http://visjs.org/dist/vis.js:13275 in Timeline
  http://visjs.org/examples/timeline/basicUsage.html:42
Status: success
```

The error suggests that `bind` is undefined on `this.redraw`, which is a function and should have a `bind` method.

**Comment by josdejong** [comment]
Ok found the cause: phantomjs seems to use an old rendering engine which doesn't have support for the ES5 function `bind` method: https://github.com/ariya/phantomjs/issues/10522

To solve the issue, you will have to load ES5-shim in your webpage (same as you would have to do if you want to use vis.js in old browsers like IE8):

```html
<!DOCTYPE HTML>
<html>
<head>
  <title>Timeline | Basic demo</title>

  <style type="text/css">
    body, html {
      font-family: sans-serif;
    }
  </style>

  <!-- load ES5 shim needed to run on old browsers IE8 and PhantomJS -->
  <script src="https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.1.10/es5-shim.min.js"></script>

  <!-- load vis.js -->
  <script src="https://cdnjs.cloudflare.com/ajax/libs/vis/4.7.0/vis.min.js"></script>
  <link href="https://cdnjs.cloudflare.com/ajax/libs/vis/4.7.0/vis.min.css" rel="stylesheet" type="text/css" />
</head>
<body>

<p>
  A basic timeline. You can move and zoom the timeline, and select items.
</p>

<div id="visualization"></div>

<script type="text/javascript">
  // DOM element where the Timeline will be attached
  var container = document.getElementById('visualization');

  // Create a DataSet (allows two way data-binding)
  var items = new vis.DataSet([
    {id: 1, content: 'item 1', start: '2014-04-20'},
    {id: 2, content: 'item 2', start: '2014-04-14'},
    {id: 3, content: 'item 3', start: '2014-04-18'},
    {id: 4, content: 'item 4', start: '2014-04-16', end: '2014-04-19'},
    {id: 5, content: 'item 5', start: '2014-04-25'},
    {id: 6, content: 'item 6', start: '2014-04-27', type: 'point'}
  ]);

  // Configuration for the Timeline
  var options = {};

  // Create a Timeline
  var timeline = new vis.Timeline(container, items, options);
</script>
</body>
</html>
```

**Comment by AlexDM0** [comment]
Isnt there a new version of phantom? PreES5 is very very old...

Is the phantom project still alive? Maybe we could do a pull?

**Comment by josdejong** [comment]
I tried with v1.9.8. There is a v2.0.0 available, which may have solved the issue. 2.0 isn't yet available on linux so I can't try, maybe you can try it on windows?

**Comment by josdejong** [comment]
phantomjs v2 indeed has an updated version of webkit, so this issue may probably be solved in 2.0

http://phantomjs.org/release-2.0.html
